.gitignore

```diff
@@ -1,5 +1,7 @@
 .DS_Store
-.pyc
+*~
+*.svg
+*.pyc
```

Comment:
I think the `.pyc` was supposed to be `*.pyc`? and the other two lines are also used for similar purpose - ignore irrelevant files that may appear during local development. Many `*~` and `*.svg` (viz output) appears in the directory when I was working on this. I guess these changes are useful? or not?